README.md

```diff
@@ -0,0 +1 @@
+# Scala Book Bot
```

Comment:
Good things to include in the README
- What the project does
- How to get it running locally
- How to run the tests